cmd/spire-agent/cli/run/run.go

```diff
@@ -180,23 +180,14 @@ func mergeConfigs(c *agent.Config, fileConfig, cliConfig *runConfig) error {
 }
 
 func mergeConfig(orig *agent.Config, cmd *runConfig) error {
-	// Parse server address
 	if cmd.AgentConfig.ServerAddress != "" {
-		ips, err := net.LookupIP(cmd.AgentConfig.ServerAddress)
-		if err != nil {
-			return err
-		}
-
-		if len(ips) == 0 {
-			return fmt.Errorf("Could not resolve ServerAddress %s", cmd.AgentConfig.ServerAddress)
-		}
-		serverAddress := ips[0]
-
-		orig.ServerAddress.IP = serverAddress
+		_, port, _ := net.SplitHostPort(orig.ServerAddress)
```

Comment:
nit: we should maybe consider deprecating `ServerAddress` and `ServerPort` in lieu of a new `ServerHost` flag or something... I recognize that we don't want to break existing configs, but moving forward, it would be convenient and maybe clean up some of the processing code to have them specified in a single value.

Reply:
Sounds reasonable.